src/components/Mapviewer.tsx

```diff
@@ -1,161 +1,129 @@
-          // console.log('hoan source', map.getSource('measuringstation'))
-          // console.log('hoan layer', map.getLayer('layer-1'))
-          // map.addImage('hoanImage', image, { sdf: true })
-          map.loadImage(
-            pumpIcon,
-            (e, img) => {
-              if (e || !img) return console.log('Failed to load image: ', e);
-              map.addImage('pumpIconImage', img, { sdf: true });
-            }
-          );
-        }}
+      <Map
+        bounds={getBounds(raster)}
       >
-        {popupData && popupData.features?.length ? (
-          <Popup
-            latitude={popupData.lngLat[1]}
-            longitude={popupData.lngLat[0]}
-            closeButton={true}
-            closeOnClick={false}
-            onClose={() => setPopupData(null)}
-            anchor="top"
-          >
-            <h3>Properties</h3>
-            {popupData.features.map((feature: any, i: number) => {
-              return (
-                <div key={i}>
-                  <hr />
-                  <h4>{feature.source}</h4>
-                  {Object.keys(feature.properties).map(key => {
-                    return (
-                      <div key={key}>
-                        {key}: {feature.properties[key]}
-                      </div>
-                    );
-                  })}
-                </div>
-              )
-            })}
-          </Popup>
-        ) : null}
-
-        {/* Vector tile layer for measuring stations from Lizard */}
-        <Source
-          key={"measuringstation"}
-          id={"measuringstation"}
-          type={'vector'}
-          tiles={[
-            `/api/v4/measuringstations/vectortiles/{z}/{x}/{y}/?organisation__uuid=${selectedOrganisation.uuid}`
-          ]}
-          minzoom={6}
-          maxzoom={14}
-        >
-          <Layer
-            key={'layer-1'}
-            id={'layer-1'}
-            // type={'circle'}
-            type={'symbol'}
-            source={'measuringstation'}
-            source-layer={'default'}
-            layout={{
-              "text-field": "{object_name}",
-              "text-size": 14,
-              "text-anchor": "bottom-left",
-              "icon-image": "pumpIconImage",
-              "icon-anchor": "bottom-right",
-              "icon-size": 0.1
+        <TileLayer
+          url={`https://api.mapbox.com/styles/v1/nelenschuurmans/ck8sgpk8h25ql1io2ccnueuj6/tiles/256/{z}/{x}/{y}@2x?access_token=${mapBoxAccesToken}`}
+        />
+        {timestamps.map(timestamp => (
+          <WMSTileLayer
+            key={timestamp.time}
+            url={'/wms/'}
+            layers={'radar:5min'}
+            styles={'radar-5min'}
+            time={timestamp.time}
+            format={'image/png'}
+            uppercase={true}
+            bounds={getBounds(raster)}
+            tileSize={1024}
+            opacity={timestamp.time === timestamps[step].time ? 1 : 0}
```

Comment:
Yes, I saw it too in the WaterDepthImageLayer component and you set the visibility in the Pane. In my case, I checked and there is no difference between using visibility and opacity (both are smooth), but that's because I preloaded all the tiles.